error: `break` outside of a loop
  --> ROOT/tests/ui/parser/break_outside_loop.sol:LL:CC
   |
LL | / function f() {
LL | |     break;
LL | | }
   | |_^
   |

error: aborting due to 1 previous error

